sci-biology/fsl/fsl-5.0.8.ebuild

```diff
@@ -108,7 +108,8 @@ src_install() {
 	sed -i "s:\$FSLDIR/tcl:/usr/libexec/fsl:g" \
 		$(grep -l "\$FSLDIR/tcl" tcl/*) || die
 
-	dobin bin/*
+	exeinto /usr/share/${PN}/bin
```

Comment:
> Why is that preferred?

Executables which aren't placed in *bin should go there.

https://refspecs.linuxfoundation.org/FHS_3.0/fhs/ch04s07.html